server/index.js

```diff
@@ -17,19 +17,19 @@ app.get('/pictures/:id', (req, res) => {
 });
 
 app.get('/information/:id', (req, res) => {
-  request.get(`http://localhost:3400${req.path}`).pipe(res);
+  request.get(`http://foodigosidebar-env.us-east-2.elasticbeanstalk.com/${req.path}`).pipe(res);
```

Comment:
I think your req.path has a slash '/' at the beginning, so if you have a slash at the end of the site, it might be two slashes in a row? Are you sure this code is loading? If it is, I guess it's fine, but worth investigating